### PR TITLE
🐽 Display information that no openings found

### LINKED ISSUE #707: Display information that there are no openings
In the Working Groups -> Openings:

*   display information when no openings were fetched similar to other "empty" states.

Follow-up for: #665

### PR DESCRIPTION
**Pull request by DevNikita16**
Closes #707

### CODE REVIEW

File: packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpenings.tsx [hunk 1]
```diff
@@ -43,7 +48,23 @@ export const WorkingGroupsOpenings = () => {
...
+    if (!(type === 'CURRENT' ? currentOpenings : upcomingOpenings).length) {
+      return <TextBig>No {type.toLowerCase()} openings found</TextBig>
+    }
+
+    return type === 'CURRENT' ? (
+      <OpeningsList openings={currentOpenings} />
+    ) : (
+      <UpcomingOpeningsList openings={upcomingOpenings} />
+    )
+  }
```
Comment (jodator) [inline packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpenings.tsx:66]: I'm not sure if this is super easy to read. Maybe something like  

```tsx
  const displayOpenings = <T extends BaseOpening>(
    loading: boolean,
    openings: T[],
    Component: JSXElementConstructor<{ openings: T[] }>
  ) => {
    if (loading) {
      return <Loading />
    }

    if (openings.length) {
      return <TextBig>No openings found</TextBig>
    }

    return <Component openings={openings} />
  }
```

then:

```tsx
{activeTab === 'OPENINGS' && displayOpenings(upcomingLoading, upcomingOpenings, UpcomingOpeningsList)}
{activeTab === 'UPCOMING' && displayOpenings(currentLoading, currentOpenings, OpeningsList)}
```

Then you can also move this function out of the main render function scope, so fewer function creations as well.

Comment (jodator) [reply packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpenings.tsx:66]: And it looks like component :upside\_down\_face: 

`<OpeningsTab isLodaing={upcomingLoading} openings={upcomingOpenings} component={UpcomingOpenings} />`

**Review by jodator — COMMENTED** [review]
I think that fewer ifs the better, WDYT?

File: packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpenings.tsx (hunk 1 above)
Comment (DevNikita16) [reply packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpenings.tsx:66]: Agree, looks just awesome!

**Pull request merged**